### ISSUE TITLE
Error in training after Training Started.

### ISSUE THREAD

**Issue by ZhaorongHan** [ildoonet/tf-pose-estimation#119]
when i training using parameters : python3 train.py --model=cmu --datapath={datapath} --batchsize=64 --lr=0.001 --modelpath={path-to-save}

there is a error after : 
```
[2018-03-14 14:13:11,065] [train] [INFO] model weights initialization
[2018-03-14 14:13:24,144] [train] [INFO] Restore pretrained weights...
[2018-03-14 14:15:11,664] [train] [INFO] Restore pretrained weights...Done
[2018-03-14 14:15:11,664] [train] [INFO] prepare file writer
[2018-03-14 14:15:15,235] [train] [INFO] prepare coordinator
[2018-03-14 14:15:15,435] [train] [INFO] Training Started.
```
```
Caused by op 'Mconv5_stage2_L2/Conv2D', defined at:
  File "/home/gy/hzr/tf-pose-estimation/src/train.py", line 99, in <module>
    net, pretrain_path, last_layer = get_network(args.model, q_inp_split[gpu_id])
  File "/home/gy/hzr/tf-pose-estimation/src/networks.py", line 36, in get_network
    net = CmuNetwork({'image': placeholder_input}, trainable=trainable)
  File "/home/gy/hzr/tf-pose-estimation/src/network_base.py", line 60, in __init__
    self.setup()
  File "/home/gy/hzr/tf-pose-estimation/src/network_cmu.py", line 55, in setup
    .conv(7, 7, 128, 1, 1, name='Mconv5_stage2_L2')
  File "/home/gy/hzr/tf-pose-estimation/src/network_base.py", line 35, in layer_decorated
    layer_output = op(self, layer_input, *args, **kwargs)
  File "/home/gy/hzr/tf-pose-estimation/src/network_base.py", line 245, in conv
    output = convolve(input, kernel)
  File "/home/gy/hzr/tf-pose-estimation/src/network_base.py", line 240, in <lambda>
    convolve = lambda i, k: tf.nn.conv2d(i, k, [1, s_h, s_w, 1], padding=padding)
  File "/home/gy/anaconda3/envs/tensorflow/lib/python3.5/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 631, in conv2d
    data_format=data_format, name=name)
  File "/home/gy/anaconda3/envs/tensorflow/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/gy/anaconda3/envs/tensorflow/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 2956, in create_op
    op_def=op_def)
  File "/home/gy/anaconda3/envs/tensorflow/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1470, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

ResourceExhaustedError (see above for traceback): OOM when allocating tensor with shape[48,128,46,46]
	 [[Node: Mconv5_stage2_L2/Conv2D = Conv2D[T=DT_FLOAT, data_format="NHWC", padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](Mconv4_stage2_L2/Mconv4_stage2_L2, Mconv5_stage2_L2/weights/read)]]
	 [[Node: Adam/update/_1150 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_8782_Adam/update", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]
```
somebody can  help me ?

**Comment by ZhaorongHan** [comment]
The problem has been solved when i reduce batch size to 32,but the loss is __nan__ and the procedure is slow , why?
```
[2018-03-14 14:33:29,321] [train] [INFO] prepare coordinator
[2018-03-14 14:33:29,519] [train] [INFO] Training Started.
[2018-03-14 14:35:46,653] [train] [INFO] epoch=0.03 step=100, 23.3349 examples/sec lr=0.010000, loss=nan, loss_ll=nan, loss_ll_paf=nan, loss_ll_heat=nan, q=13
[2018-03-14 14:38:38,064] [train] [INFO] epoch=0.05 step=200, 20.7426 examples/sec lr=0.010000, loss=nan, loss_ll=nan, loss_ll_paf=nan, loss_ll_heat=nan, q=2
[2018-03-14 14:41:59,817] [train] [INFO] epoch=0.08 step=300, 18.8126 examples/sec lr=0.010000, loss=nan, loss_ll=nan, loss_ll_paf=nan, loss_ll_heat=nan, q=2
[2018-03-14 14:45:13,738] [train] [INFO] epoch=0.11 step=400, 18.1762 examples/sec lr=0.010000, loss=nan, loss_ll=nan, loss_ll_paf=nan, loss_ll_heat=nan, q=2
[2018-03-14 14:48:30,964] [train] [INFO] epoch=0.13 step=500, 17.7493 examples/sec lr=0.010000, loss=nan, loss_ll=nan, loss_ll_paf=nan, loss_ll_heat=nan, q=2
[2018-03-14 14:51:37,954] [train] [INFO] epoch=0.16 step=600, 17.6400 examples/sec lr=0.010000, loss=nan, loss_ll=nan, loss_ll_paf=nan, loss_ll_heat=nan, q=2
```

**Comment by ouceduxzk** [comment]
decrease lr, 0.0001 or smaller

**Comment by ZhaorongHan** [comment]
thank you very much ,it's ok now!
